cishouseholds/pipeline/high_level_transformations.py

```diff
@@ -441,6 +441,10 @@ def transform_survey_responses_version_digital_delta(df: DataFrame) -> DataFrame
         {k: dont_know_mapping_dict for k in dont_know_columns},
     )
 
+    if "self_isolating_reason" in df.columns:
```

Comment:
This has been accidentally included when resolving conflicts with main, please remove it